libstb/secvar/backend/edk2-compat.c

```diff
@@ -163,10 +165,16 @@ static int edk2_compat_process(void)
 
 static int edk2_compat_post_process(void)
 {
+	struct secvar_node *hwvar;
 	printf("setup mode is %d\n", setup_mode);
 	if (!setup_mode) {
 		secvar_set_secure_mode();
 		prlog(PR_INFO, "Enforcing OS secure mode\n");
+		hwvar = find_secvar("HWKH", 5, &variable_bank);
```

Comment:
Probably should have a short comment explaining why we are doing this: "hw-key-hash is no longer needed after this point, and doesn't need to be exposed"